compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/function-expression-prototype-call.js

```diff
@@ -0,0 +1,11 @@
+function Component(props) {
+  const f = function () {
+    return <div>{props.name}</div>;
+  };
+  return f.call();
```

Comment:
failed previously because `f` had the inferred type BuiltInFunction which didn't have a shape defined